config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,10 @@
+if Rails.env.production? && Rails.configuration.kubernetes_deployment?
+  Sidekiq.configure_server do |config|
+    config.redis = { url: "redis://#{Rails.configuration.redis_url}:6379", network_timeout: 5 }
```

Comment:
We are currently getting `connection lost` errors with Redis. Possibly a Cloud Platform issue

Reply:
Is it worth also specifying the reconnection config as in https://github.com/redis/redis-rb#reconnections